conf/messages

```diff
@@ -23,8 +23,11 @@ global.opening.times.weekend.p=Closed weekends and bank holidays.
 global.opening.times.bankholidays.p=Closed on bank holidays.
 global.opening.times.24_7=24 hours a day, 7 days a week
 global.closed.time.sun.banckholidays=Closed Sunday and bank holidays.
-global.opening.time.banckholidays.8to4=Open 8am to 4pm on bank holidays.
+global.opening.time.banckholidays.8to4=Bank Holidays, 8am to 4pm.
+global.opening.time.sat.bankholidays.8to4=Saturdays and Bank Holidays, 8am to 4pm
 global.closed.time.sun=Closed Sundays
+global.closed.time.sun.fullstop=Closed Sundays.
+global.closed.time.bankholidays=Closed bank holidays.
```

Comment:
Do we need "global.closed.time.sun.fullstop" ? 
Think we should just add a full stop to the existing message. for consistency:
global.closed.time.sun=Closed Sundays.